kotkt.nl/content/_index.nl.md

```diff
@@ -60,12 +60,12 @@ Wil je de sfeer proeven? Kom dan vooral een keer op onze bord- of rollenspelavon
 {{< aside >}}
    {{< banner rotate="6" padding="41px">}}
        {{< figure src="images/Rollenspellen_icons.png" >}}
-       One-shot rollenspelavond
-       Elke 1e woensdag van de maand
+       **One-shot rollenspelavond: **
```

Comment:
Helaas kun je geen Markdown gebruiken in een aside block, dus je moet terugvallen op HTML:
```suggestion
       <b>One-shot rollenspelavond:</b>
```

Reply:
oeh suggesties zijn ook een ding in GitHub. Fancy!